### PR TITLE
CORE-1147 pipelines dispatch

### PR DESCRIPTION
**Pull request by ellisonc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+jobs:
+  dispatch_job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run Pipelines Dispatch
+        uses: path-to-your-action-repo@v1
```
Comment (MoonMoon1919) [inline README.md:37]: Should this be `gruntwork-io/pipelines-dispatch@v1` or something similar?

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    description: "The directory in which Terragrunt will run the action"
+    required: true
+  command:
+    description: "The command (e.g., plan, apply, etc) for Terragrunt to run"
+    required: true
+  args:
+    description: "The arguments to pass into Terragrunt"
+    required: true
+  token:
+    description: "GitHub PAT to clone the pipelines repo"
```
Comment (MoonMoon1919) [inline action.yml:26]: maybe...
```suggestion
    description: "GitHub PAT which has access to your infrastructure-live and infrastructure-pipelines repositories"
```

**Review by MoonMoon1919 — COMMENTED** [review]
Two small comments

File: README.md (hunk 1 above)
Comment (ellisonc) [reply README.md]: yep yep, that's what happens when you make chat gpt write your readme's for you :)

**Review by MoonMoon1919 — APPROVED** [review]

**Review by MoonMoon1919 — APPROVED** [review]

**Pull request merged**